### ISSUE TITLE
IndexError when calling Mesh.from_file()

### ISSUE THREAD

**Issue by gustxsr** [sfepy/sfepy#537]
Hello!
I am trying to create an instance of Mesh with a file that contains the mesh of a 2D surface that is curved into the shape of multiple spheres joined together. However, when I run:
`mesh = Mesh.from_file(my_file)`
I get an IndexError message (I attached a picture of the full message in this post). Also, before the error message, the program was outputting multiple lines such as:
`sfepy:     skipping unknown entity: ['0.03790829218172548', '-0.07941428261762333', '-0.3106061323349568']`
But with different coordinates or numbers every time. Could you help me figure out what went wrong?
Thank you beforehand!
Gustavo Santiago-Reyes

![Error message](https://user-images.githubusercontent.com/46436777/61446410-1603bc00-a91d-11e9-9dc6-38b661662e2c.png)

**Comment by vlukes** [comment]
Hi Gustavo,

could you send me the mesh file (surface.msh)?

Regards
Vladimir

**Comment by gustxsr** [comment]
Thank you for your time and answer! Initially, my file was .unv format and I changed this extension to .mesh (by deleting/typing it) and I believe that that's the reason why it didn't work. After some time trying different approaches, I found that running:
`gmsh -3 -format mesh -string 'Mesh.SaveElementTagType=2;' surface.unv`
Correctly changes the format of my mesh file and allows me to create the Mesh instance.  I do have another question that I haven't figured out, but I will open another discussion because it's of a different topic that arose from this one. 
-Gustavo
